src/web/lib/components/ThemeSaveButton/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import './index.scss';
+
+export const ThemeSaveButton = ({
+  theme,
+  savedThemes,
+  generateThemeKey,
+  putTheme
+}) => {
+  const saveTheme = () => putTheme(generateThemeKey(), theme);
```

Comment:
If we want to use user-supplied names in the UX, we can switch from this randomly generated key to something from a text field.